### DISCUSSION TITLE
New source: Samsung Galaxy Store?

### DISCUSSION THREAD

**Discussion by DwainZwerg** [ImranR98/Obtainium#1418] [Ideas]
I would actually love to have the Galaxy Store as a source. Because I only have one app left (Find my mobile) from Samsung, which is neither in end of support for my device nor uninstalled by me, I uninstalled the Samsung Galaxy Store some time ago and update Find my mobile with apkmirror.com instead (track-only in Obtanium). I used to think that you can't add this source because Samsung doesn't even offer a .js download button [on their site](https://galaxystore.samsung.com/apps). However, I have now realized that there is a [Developer API ](https://developer.samsung.com/galaxy-store/galaxy-store-developer-api.html). It looks like, in principle at least, you can also retrieve data about apps, such as when an app was last updated. The only problem is that these APIs are designed more for developers and are used more for managing your own published apps.

Is there any chance of adding the Galaxy Store (if not otherwise possible as track-only)?

**Comment by DwainZwerg** [comment]
I just realized: You can even install/update an app if you enter your Samsung account: https://galaxystore.samsung.com/prepost/000004700911?appId=com.samsung.android.fmm

**Comment by 8pen-s8urce** [comment]
I think that this should be opened as a feature request (for a new app source) in Obtainium's issue tracker.

**Comment by inson1** [comment]
@DwainZwerg btw dont forget to close it, when you did fr from it :)
